src/style.css

```diff
@@ -1 +1,141 @@
 /* add styles here */
+body {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: 900;
+  font-style: normal;
+}
+
+
+.search-bar {
+  display: flex;
+  align-items: center;
+  background-color: white;
+  width: calc(100% - 2px);
+  height: 68px;
+  position: relative;
+  margin-top: 20px;
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
+  color: #3d4e61;
+}
+
+.search-bar:focus-within {
+  background: linear-gradient(180deg, #FFF 0%, #F6F6F7 100%);
+}
+
+.search-bar:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search-bar .search-bar__icon {
+  background-image: url(./images/Search.svg);
+  width: 19px;
+  height: 19px;
+  background-repeat: no-repeat;
```

Comment:
pls try to combine `background-` properties into one `background:` property
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**